udata_gouvfr/theme/templates/footer.html

```diff
@@ -1,3 +1,4 @@
+{% cache 60*5, 'footer', g.lang_code|default(config.DEFAULT_LANGUAGE) %}
```

Comment:
No, it's just for the snippet used in error pages.
Another alternative is to set the value for error pages.